tools/code-generator/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.11.0b5-alpine3.15 as base
+FROM python:3.9.18-alpine3.18 as base
```

Comment:
@etiennecallies what was the reasoning for changing the code generation Docker base image as well? It seems that your problem would be solved solely by changing `airbyte-cdk/python/Dockerfile`,  right?

Reply:
Hi @maxi297! You're absolutely right. I wasn't sure about this. This PR is more a friendly "hi guys we should upgrade alpine" than "please merge this". Thanks